common/src/test/java/com/revenuecat/purchases/common/BillingResponseCodeTest.kt

```diff
@@ -0,0 +1,65 @@
+package com.revenuecat.purchases.common
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.android.billingclient.api.BillingClient
+import com.revenuecat.purchases.PurchasesErrorCode
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class ErrorUtilsTests {
+
+    @Test
+    fun `There are no new BillingResponseCodes`() {
+        val allPossibleBillingResponseCodes = BillingClient.BillingResponseCode::class.java.declaredFields
+        assertThat(allPossibleBillingResponseCodes)
+            .withFailMessage("It looks like there are new BillingResponseCodes, " +
+                "make sure to update the error conversions functions to include the new errors.")
+            .hasSize(12)
+    }
+
+    @Test
+    fun `All BillingResponseCodes are handled when converting to PurchasesError`() {
+        val allPossibleBillingResponseCodes = BillingClient.BillingResponseCode::class.java.declaredFields
+        allPossibleBillingResponseCodes.forEach { field ->
+            val value = field.getInt(field)
+            val billingResponseToPurchasesError = value.billingResponseToPurchasesError("")
+            if (value != BillingClient.BillingResponseCode.OK) {
+                assertThat(billingResponseToPurchasesError.code)
+                    .withFailMessage("BillingClient.BillingResponseCode.%s is unhandled", field.name)
+                    .isNotEqualTo(PurchasesErrorCode.UnknownError)
+            } else {
+                assertThat(billingResponseToPurchasesError.code).isEqualTo(PurchasesErrorCode.UnknownError)
+            }
```

Comment:
I couldn't come up with a better way of testing this, but I think this is good enough.